### PR TITLE
add bionic/cosmic for python-click, python-skimage, python-sklearn

### PR DESCRIPTION
**Pull request by k-okada**
https://packages.ubuntu.com/bionic/python-sklearn
https://packages.ubuntu.com/bionic/python-skimage
https://packages.ubuntu.com/bionic/python-click

### CODE REVIEW

**Review by tfoote — DISMISSED** [review]

File: rosdep/python.yaml [hunk 1]
```diff
@@ -761,6 +761,9 @@ python-click:
     pip:
       packages: [click]
   ubuntu:
+    artiful: [python-click]
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: typo: artiful -> artful, same below

**Comment by mikaelarguedas** [comment]
I moved to the wildcard approach to fix the artful typo and make it more futureproof.
@tfoote could you please rereview this ? thanks!

**Comment by mikaelarguedas** [comment]
This will likely conflict with https://github.com/ros/rosdistro/pull/18634 that contains similar changes

**Review by tfoote — APPROVED** [review]
This lgtm and looks like a superset of #18634

**Pull request merged**